apiserver/addresser/addresser_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/juju/juju/apiserver/common"
 	"github.com/juju/juju/apiserver/params"
 	apiservertesting "github.com/juju/juju/apiserver/testing"
-	"github.com/juju/juju/cmd/envcmd"
+	"github.com/juju/juju/cmd/basecmd"
```

Comment:
Did we really need to change the package name? "envcmd" seemed to make sense to me and leaving it the same would reduce the churn somewhat.